addons/website/models/assets.py

```diff
@@ -74,7 +74,7 @@ def make_scss_customization(self, url, values):
                 else:
                     font_family_attachments = IrAttachment
                     font_content = requests.get(
-                        f'https://fonts.googleapis.com/css?family={font_name}&display=swap',
+                        f'https://fonts.googleapis.com/css?family={font_name}:300,300i,400,400i,700,700i&display=swap',
```

Comment:
This commit should not be part of the PR, looks like a rebase error

Reply:
No it should not, looks like mistakenly @Guillaume-gdi pushed his commit on this branch.

I guess I can remove it?

Reply:
![image](https://github.com/odoo/odoo/assets/78849981/688a0197-94af-42a9-a7b3-cec71b57c241)
I'm the author of the commit but I didn't committed it here

Reply:
okay must be my mistake then